.gitignore

```diff
@@ -10,3 +10,5 @@ tmp
 .classpath
 .project
 .settings
+*.swp
+tags
```

Comment:
What’s a swp file?

Tags seems like something that maybe belongs in ~/.config/git/ignore (because it’s relevant to user practice, rather than the specific project).